trust-quorum/src/connection_manager.rs

```diff
@@ -388,6 +380,45 @@ impl ConnMgr {
         }
     }
 
+    // After we have updated our network config, we should send it out to all
+    // peers with established connections, with the exception of the peer we
+    // received it from if this was not a local update.
+    pub async fn broadcast_network_config(
+        &mut self,
+        network_config: &NetworkConfig,
+        excluded_peer: Option<&BaseboardId>,
+    ) {
+        for h in self
+            .established
+            .iter()
+            .filter(|&h| Some(&h.baseboard_id) != excluded_peer)
+        {
+            info!(
+                self.log,
+                "Sending network config";
+                "peer_id" => %h.baseboard_id,
+                "generation" => network_config.generation
+            );
+            h.task_handle.send_network_config(network_config.clone()).await;
+        }
```

Comment:
Can we call `send_network_config` instead of duplicating its contents here? Otherwise if we change one function we have to remember to change the other function too, and at some point we will forget.